ludwig/models/llm.py

```diff
@@ -106,14 +145,25 @@ def __init__(
         )
 
         # Extract the decoder object for the forward pass
-        _, self.output_feature_decoder = self.output_features.items()[0]
-
-        # ================ Combined loss metric ================
-        self.eval_loss_metric = torchmetrics.MeanMetric()
-        self.eval_additional_losses_metrics = torchmetrics.MeanMetric()
+        self._output_feature_decoder = ModuleWrapper(self.output_features.items()[0][1])
 
         clear_data_cache()
 
+    def create_feature_dict(self) -> LudwigFeatureDict:
+        return DictWrapper(LudwigFeatureDict())
+
+    @property
+    def eval_loss_metric(self) -> torchmetrics.MeanMetric:
+        return self._eval_loss_metric.module
+
+    @property
+    def eval_additional_losses_metrics(self) -> torchmetrics.MeanMetric:
+        return self._eval_additional_losses_metrics.module
```

Comment:
Do we need these if they're already defined in the base model class?

Reply:
Good catch! Removed.